[UIK-2988] Start/stop button and notification in SpinContainer, Spin, Skeleton examples

sheila-semrush opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Motivation and Context

Added start/stop button to examples of Spin, Skeleton, SpinContainer, instead of constant automatic switching.
Also added live messages on loading start/stop wherever needed.

<img width="647" alt="image" src="https://github.com/user-attachments/assets/50ea65eb-4b00-4e50-a7f6-1035c34126a3" />

## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Nice improve.

## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly or it's not required.
- [ ] Unit tests are not broken.
- [ ] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

ilyabrower commented on stories/components/data-table/docs/examples/skeleton-in-table.tsx:
```diff
@@ -16,26 +18,42 @@ function getSkeleton() {
+          {...(loading ? { rows: [getSkeleton(), getSkeleton(), getSkeleton()] } : {})}
+        />
+      </DataTable>
+      <Button onClick={handleLoading} mt={3}>
```
just a small nitpick - on my part it would be more correct to say `toggleLoading` =)

sheila-semrush replied:
no problem, fixed, thanks!

ilyabrower commented on website/docs/table-group/data-table/data-table-code.md:
```diff
@@ -80,18 +80,6 @@ Scroll in the table header is useful for very long tables with fixed columns, al
-::: sandbox
-
-<script lang="tsx">
-  export Demo from 'stories/components/data-table/docs/examples/fixed-header-with-spin-overlay.tsx';
```
let's remove this example from storybook too

j-mnizhek commented on website/docs/components/input/examples/loading_state_in_the_input.tsx:
I have a proposal to update the placeholder for the input too, because it uses word "see" and the whole sentence is funny, but has no actual meaning: "Type something to see world spinning..." — to something like: "Type something to start loading state..." or something else.

j-mnizhek reviewed (approved):
Overall looks and reads good 😎

sheila-semrush commented:
@j-mnizhek @Valeria-Zimnitskaya
found one more example that needed a Restart button: [ProgressBar](http://localhost:5173/intergalactic/components/progress-bar/progress-bar-code#basic-usage)
Bar is focused on Restart to notify the user about the state change
also updated aria-labels

sheila-semrush commented:
@j-mnizhek also in ProgressBar: removed the animation section, because it was incorrect (our animation is linear), and I'm not sure it's needed. Moved one section, to the motive of our recent heading structure changes.
